middlewares/index.js

```diff
@@ -10,32 +10,36 @@ const env = process.env.NODE_ENV || 'development';
 const config = require('../config/config.json')[env];
 
 const utilityService = require('../services/utilityService');
+const logger = require('../modules/logger');
 
 module.exports = {
 
     logRequest: function(req, res, next) {
-    	if (req.method == 'POST') {
-    	    const ipAddress = (
-    	                req.headers['x-forwarded-for'] 
-    	                        && req.headers['x-forwarded-for'].split(',').pop())
-    	            || req.connection.remoteAddress
-    	            || req.socket.remoteAddress
-    	            || req.connection.socket.remoteAddress;
+    	const ipAddress = (
+    	            req.headers['x-forwarded-for'] 
+    	                    && req.headers['x-forwarded-for'].split(',').pop())
+    	        || req.connection.remoteAddress
+    	        || req.socket.remoteAddress
+    	        || req.connection.socket.remoteAddress;
 
-    	    const url = req.originalUrl;
-    	    const body = req.body;
+    	const url = req.originalUrl;
+    	const body = req.body;
 
-    	    console.log('POST IPAddress: ', ipAddress);
-    	    console.log('\x1b[36m%s\x1b[0m', 'Request URL:', url);
-    	    console.log('POST: ', body);
-    	    console.log('\x1b[33m%s\x1b[0m', '-----------------------------');
+    	logger.info('IPAddress: ' + ipAddress);
+    	logger.info('Request Method: ' + req.method);
+    	logger.info('Request Url: ' + url);
+    	logger.info('Request Body: ', body);
```

Comment:
We should probably use a format for logging requests, so that it'll be easier for us to query logs later on with the help of some script

Here is one example
127.0.0.1  [10/Oct/2000:13:55:36 -0700] "GET /apache_pb.gif HTTP/1.0" 200 2326

IP.              Time                                             method and path         response_status_code      response_size

Reply:
i tried morgan printing same format as mentioned but then dropped it. This format is valid only for requestLogs and not others. Critical one i am already saving to db. It's your call though

Reply:
Ok, we should merge all these info into one for now.

Reply:
yeah but then that would be unreadable manually (body data or urls can be too long). I'll look if printing one big line can be formatted by winston.

Reply:
I think we need not print the body?

Reply:
yeah we can remove them from production environment. We'll put again (except personal data) in future if needed.